ansible/app_options_files/empty

```diff
@@ -0,0 +1 @@
+
```

Comment:
Agreed, although I was thinking separate genesis.json files for different applications. This is actually quite well managed in ethermint: the startup genesis.json is part of the source code repository and the package just puts it in place. Maybe we should do something similar with basecoin: have multiple genesis.json files pre-generated as examples and during installation the right one is copied over to the config folder. I don't really see the reason to manage genesis.json files for all kinds of applications here.